requirements.txt

```diff
@@ -1,4 +1,5 @@
-libpebble2==0.0.26
+#libpebble2==0.0.28
+git+https://github.com/pebble-dev/libpebble2.git@685cd9841e647b48609ce904a87540019390b3be#egg=libpebble2
```

Comment:
The libpebble2 on pypi [isn't unmaintained](https://pypi.org/project/libpebble2/) - I'd rather use that.

Reply:
Ahhh, that's not clear from the pypi page. All the links are to the old pebble project and CI. Thanks for making that clear :-). I was wondering maybe  I can open a PR to clean that up, but the readme at https://github.com/pebble-dev/libpebble2/blob/master/README.rst doesn't match the (long) description over at https://pypi.org/project/libpebble2/ - I'll get a bug opened over there but notes (mostly to myself)

https://pypi.org/project/libpebble2/ points to:

* https://github.com/pebble/libpebble2 (multiple places) - instead of https://github.com/pebble-dev/libpebble2
* https://travis-ci.org/pebble/libpebble2 - which also points to https://github.com/pebble/libpebble2

I don't remember why I felt the direct link was necessary, I'm reasonably confident I was ending up with the wrong/old version and it didn't work without it.

Reply:
Issues are not enabled on https://github.com/pebble-dev/libpebble2 - https://pypi.org/project/libpebble2/ looks abandoned (readme/description and links are not the current readme in https://github.com/pebble-dev/libpebble2). This may be a pypi issue?

Reply:
...
> I don't remember why I felt the direct link was necessary, I'm reasonably confident I was ending up with the wrong/old version and it didn't work without it.

lol, I missed the setup.py requirements - that's probably why :-D